.github/workflows/pika.yml

```
@@ -188,6 +188,7 @@ jobs:
 
       - name: Cleanup
         run: |
+          cp deps/lib/libz.1.dylib . 
```

yamllint

[error] trailing spaces

(trailing-spaces)